lib/agenda/cancel.ts

```diff
@@ -1,16 +1,17 @@
 import { Agenda } from '.';
 import createDebugger from 'debug';
+import { FilterQuery } from 'mongodb';
 
 const debug = createDebugger('agenda:cancel');
 
 /**
  * Cancels any jobs matching the passed MongoDB query, and removes them from the database.
  * @name Agenda#cancel
  * @function
- * @param {Object} query MongoDB query to use when cancelling
+ * @param FilterQuery<any> query MongoDB query to use when cancelling
```

Comment:
Not sure if this is needed, in general when all types are in place, a documentation can be generated directly from the typings. No need for extra markups. But it will not harm too, therefore just wanted to point it out ;-)

Reply:
oh, I didn't know that.
So the generation will derive the type correctly?

That change actually was not supposed to in this PR.
I am working on a type-enhancement PR.

But thanks for letting me know.

Reply:
Yop :) which also helps a lot to keep documentation up2date :) i have used this approach in the hokfiy fork, e.g. This https://hokify.github.io/agenda/agenda/4.x/index.html documentation uses all the typescript types

Reply:
[Docs](https://agenda.github.io/agenda/) are currently generated based on these, but we can of course change that, or remove those docs entirely.